provider/cmd/pulumi-resource-frontegg/pulumi-resource-xyz/main.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
this also seems like an extra / unneeded file from the template? you already have a `provider/cmd/pulumi-resource-frontegg/main.go` defined